src/modules/dashboard/service/dashboardService.ts

```diff
@@ -835,8 +838,15 @@ export class DashboardService {
 
                 if(valueType === "asset"){
                     amountTip = DashboardService.createAssetAmountTip(newTransfer.amount, newTransfer.value, newTransfer.valueDisplay, true);
-                }
-                else{
+
+                    // get other assets
+                    if(mosaicIdHex.toUpperCase() != nativeTokenAssetId.value.toUpperCase() && mosaicIdHex.toUpperCase() != nativeTokenNamespaceId.value.toUpperCase()){
+                        (async() => {
+                            let otherAsset = await DashboardService.displayOtherAsset(newTransfer.amount, newTransfer.value, transferTx.mosaics[i].id);
```

Comment:
why use async await ?

Reply:
Because displayOtherAsset is written in async await previously to get namespace of asset from REST

Reply:
as you changed the code, there is nothing to wait, so no reason to use async - await